### PR TITLE
added linux arm64 for extra use cases and changed amd64 to x86_64 for…

### PR DESCRIPTION
**Pull request by Wyvinar**
… cli continuity

### CODE REVIEW

File: openapi2beans/Makefile [hunk 1]
```diff
@@ -31,15 +31,18 @@ openapi2beans-source : \
 	./pkg/utils/*.go \
 	./pkg/errors/*.go 
 
-bin/openapi2beans-linux-amd64 : openapi2beans-source
+bin/openapi2beans-linux-x86_64 : openapi2beans-source
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/openapi2beans-linux-amd64 ./cmd/openapi2beans
```
Comment (techcobweb) [inline openapi2beans/Makefile:35]: arm64 mentioned. Should match the thing which is the build target.

File: openapi2beans/Makefile [hunk 2]
```diff
@@ -31,15 +31,18 @@ openapi2beans-source : \
...
 
-bin/openapi2beans-darwin-amd64 : openapi2beans-source
+bin/openapi2beans-darwin-x86_64 : openapi2beans-source
 	CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -o bin/openapi2beans-darwin-amd64 ./cmd/openapi2beans
```
Comment (techcobweb) [inline openapi2beans/Makefile:38]: same here

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-fnmh5 for details. If you are unable to do so, please contact a member of the Galasa team.

**Review by techcobweb — APPROVED** [review]
Assuming you fix those problems. Approved.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-7xlxj for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-g4p4k for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-rbg6m for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-5kg6x for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-l2hzb for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by Wyvinar** [comment]
empty commits to see if the changes from automation have triggered yet

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-65gm5 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-g4lkf for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-688qt for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-2vpwp for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-buildutils-pr-43-nps8q for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**